rust/chains/hyperlane-cosmos/src/mailbox.rs

```diff
@@ -84,8 +90,16 @@ impl Mailbox for CosmosMailbox {
         let branch = response
             .branch
             .iter()
-            .map(|b| H256::from_slice(&hex::decode(b).unwrap()))
-            .collect::<Vec<H256>>();
+            .map(|b| {
+                if b.is_empty() {
+                    "0000000000000000000000000000000000000000000000000000000000000000"
+                } else {
+                    b
+                }
+            })
```

Comment:
this is unexpected

Reply:
u means the `hash result` of `0x00000..000` right? @yorhodes